SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -136,6 +145,110 @@ void FLocalDeploymentManager::RefreshServiceStatus()
 	});
 }
 
+bool FLocalDeploymentManager::CheckIfPortIsBound(int32 Port)
+{
+	ISocketSubsystem* SocketSubsystem = ISocketSubsystem::Get(PLATFORM_SOCKETSUBSYSTEM);
+	bool bSuccess = false;
+
+	// Set our broadcast address
```

Comment:
```suggestion
	// Set our broadcast address.
```
Comments end with full stop.